packages/smooth_ui_library/lib/widgets/smooth_listTile.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+
+class SmoothListTile extends StatelessWidget {
+
+  const SmoothListTile({@required this.text, @required this.onPressed, this.Widgeticon});
+
+  final String text;
+  final Widget Widgeticon;
```

Comment:
widgetIcon, or simply icon :wink:

Reply:
I didn't want to call it an icon to avoid the misconception that it has to be an icon. Now it's called trailingWidget. Trailing is probably better known to the most anyways.

Thank you for your critical look at my code and the suggestions for improvement.🙈

Reply:
Hi thanks for addressing all my comments. Looking forward to work together, I will try to free more time this coming year to work on this project so we can really get it going at a good pace.
Welcome to Open Food Facts 😉

PS: I still need to pull the code and try it before merging, I will keep you updated.